hive-metastore/src/main/java/org/apache/iceberg/hive/HiveSchemaUtil.java

```diff
@@ -166,6 +166,8 @@ private static String convertToTypeString(Type type) {
         return "timestamp";
       case FIXED:
       case BINARY:
+      case GEOMETRY:
+      case GEOGRAPHY:
```

Comment:
Why is this included? I don't think that we want to translate these into Hive types.

Reply:
I've reverted this change. Does it mean that Hive catalog won't support geospatial columns?

Reply:
When this is needed, we can add it. But there's no value in translating to binary right now.